internal/rbd/rbd_journal.go

```diff
@@ -325,7 +325,7 @@ func (rv *rbdVolume) Exists(ctx context.Context, parentVol *rbdVolume) (bool, er
 	}
 
 	if parentVol != nil && parentVol.isEncrypted() {
-		err = parentVol.copyEncryptionConfig(&rv.rbdImage, false)
+		err = parentVol.copyEncryptionConfig(&rv.rbdImage, true)
```

Comment:
Destination pvc in case of restore/clone will have possibly different encryption config already initialized.